local/test-binary-parallel-mtr

```diff
@@ -288,14 +288,21 @@ if [[ "${CI_FS_MTR}" = 'yes' ]]; then
         MTR_ARGS_CI+=" --big-test"
     fi
 
-    CI_TESTS=""
-    for CI_TESTS_TMP in $(grep --exclude="*.inc" --exclude="*.log" -rl . -e include/have_case_insensitive_file_system.inc | awk -F '/' '{print $NF}' | sed 's/.test//g'); do
-        CI_TESTS+=" $CI_TESTS_TMP"
-    done
+    # Collect all CI_FS tests
+    CI_TESTS=$(grep --exclude="*.inc" --exclude="*.log" -rl . -e include/have_case_insensitive_file_system.inc | awk -F '/' '{print $(NF-2)"."$NF}' | sed 's/\.test//g' | sed 's/^\./main/g')
     # Requested in PS-7602
-    if [[ ${CMAKE_BUILD_TYPE} = 'Debug' ]]; then
-        CI_TESTS+=" information_schema.i_s_schema_definition_debug"
-    fi
+    CI_TESTS+=" information_schema.i_s_schema_definition_debug"
```

Comment:
Actually, I dropped `if` by omission. It makes no difference as MTR test has source `include/have_debug.inc`, so it will be skipped by MTR framework anyway.
Changed back to `if debug` version.